sstables/mx/writer.cc

```diff
@@ -26,6 +26,8 @@
 #include <boost/range/adaptor/indexed.hpp>
 #include <boost/range/algorithm/stable_partition.hpp>
```

Comment:
> in addition to the old Index.db.

Should stop once the cluster is fully upgraded.

Reply:
Maybe. This part of the draft is just hacked in so far anyway. It got the least amount of attention.

I think that we want the choice of index to be configurable (per-table, probably).